benefits/core/templates/core/includes/form.html

```diff
@@ -1,38 +1,53 @@
-
+{% load i18n %}
 {% if form.action_url %}
 
   {% url form.action_url as form_action %}
 
   <form action="{{ form_action }}" method="{{ form.method | default:"post" | upper }}" role="form">
     {% csrf_token %}
 
-    {% for field in form %}
-      <div class="form-group">
-        {% if field.label %}
-          <label for="{{ field.id_for_label }}" class="form-control-label h2">
-            {{ field.label }}
-            {% if field.field.required %}<span class="required-label">*</span>{% endif %}
-          </label>
-        {% endif %}
+    <div class="row form-container">
+      <div class="{{ form.classes }}">
+        {% for field in form %}
+          <div class="row form-group mb-0">
+            <div class="col-12">
+              {% if field.label %}
+                <label for="{{ field.id_for_label }}" class="form-control-label">
+                  {{ field.label }}
+                  {% if field.field.required %}<span class="required-label text-body">*</span>{% endif %}
+                </label>
+              {% endif %}
 
-        {{ field }}
+              {{ field }}
 
-        {% if field.help_text %}<small class="form-text text-muted">{{ field.help_text }}</small>{% endif %}
+              {% if field.help_text %}<small class="form-text text-body">{{ field.help_text }}</small>{% endif %}
+            </div>
+          </div>
+        {% endfor %}
       </div>
-    {% endfor %}
+    </div>
 
     {% if form.submit_value %}
-      <div class="form-group">
-        {% if recaptcha.enabled %}
-          <button class="btn btn-lg btn-primary g-recaptcha"
-                  data-sitekey="{{ recaptcha.site_key }}"
-                  data-callback="onSubmit"
-                  data-action="submit">
-            {{ form.submit_value }}
-          </button>
-        {% else %}
-          <button class="btn btn-lg btn-primary">{{ form.submit_value }}</button>
+      <div class="row d-flex flex-column-reverse justify-content-lg-end flex-lg-row" style="padding-top: 64px;">
```

Comment:
Not a dealbreaker, but I think you created a utility class for this large padding top right?

Reply:
Good catch!